src/components/strategies/overlapping/OverlappingStrategyGraph.tsx

```diff
@@ -168,9 +168,18 @@ export const OverlappingStrategyGraph: FC<Props> = (props) => {
   const { quote, order0, order1, spread } = props;
   const baseMin = Number(formatNumber(order0.min));
   const baseMax = Number(formatNumber(order1.max));
-  const xFactor = getXFactor(baseMin, baseMax, +props.marketPrice);
 
-  const marketPrice = +props.marketPrice * xFactor;
+  const isPriceAvailable = !!props.externalPrice || !!props.marketPrice;
+
+  const actualMarketPrice = Number(
+    isPriceAvailable
+      ? props.marketPrice ?? '0'
+      : (baseMax - baseMin) / 2 + baseMin
```

Comment:
Maybe use the mean function instead
```suggestion
      : (baseMax + baseMin) / 2
```